lib/usd/ui/layerEditor/mayaSessionState.cpp

```diff
@@ -282,7 +282,8 @@ MayaSessionState::loadLayersUI(const QString& in_title, const std::string& in_de
         return std::vector<std::string>();
     else {
         std::vector<std::string> results;
-        for (const auto& file : files) {
+        for (uint32_t i = 0; i < files.length(); i++) {
```

Comment:
Build fix for Maya 2018

Reply:
Strange this one doesn't work. Don't we use this kind of for-loop all over the code?

Reply:
We don't have the necessary interfaces for range-based iteration in Maya 2018.